core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -186,20 +187,23 @@ static paddr_t core_mmu_get_ul1_ttb_pa(void)
 	return pa;
 }
 
-static void *core_mmu_alloc_l2(struct tee_mmap_region *mm)
+static void *core_mmu_alloc_l2(size_t size)
```

Comment:
Note: this must be rewritten. it is overcomplex. A simple xlat++ as in lpae.c is far sufficient.